src/util/MmapVector.h

```diff
@@ -224,11 +241,15 @@ class MmapVector {
   // invalidates iterators
   void unmap();
 
+  // advise the kernel to use a certain access pattern
+  void advise(AccessPattern pattern);
```

Comment:
what's the difference between `advice()` and `setAccessPattern()` seems a bit arbitrary